sections/introduction.tex

```diff
@@ -2,36 +2,44 @@ \section{Introduction}
-We have chosen to explore using PageRank along with topic information, since it is a popular way to rank documents.
-PageRank was originally used for ranking web pages in search engines, but it can easily be extended to other types of data, such as news articles.
-A more specific implementation of PageRank is a cluster-based PageRank\cite{ClusterPageRank}.
-This method uses similarities between sentences, clusters, and a corpus, to create better summaries of texts.
-It utilizes these three similarity functions to introduce new information between sentences in a document and combines these into one score for a sentence.
 
-We want to investigate whether the cluster level information of topics can be combined into a PageRank to rank documents based on their topics.
-Topic themes yield important information between sentences in \cite{ClusterPageRank} and we believe that this concept can be applied to articles by clustering according to topics given by \gls{lda}.
+Ranking articles can be done multiple ways, where PageRank is commonly used\cite{google_pagerank2006}.
+PageRank\cite{pagerank_1999} is a ranking algorithm, which is based on the random walk process.
+The principle is to let a number of surfers walk between nodes in a graph, where each surfer takes one step in each iteration.
+After an appropriate number of steps, the number of surfers on each node will be the ranking of the given node, where more surfers give a higher rank.
+The edges in the graph can be based on many interactions, such as hyperlinks, similarities, genres, etc.
+
+In \cite{yang2009topic} they investigate how to combine a topic model \gls{lda} and a PageRank.
+They also investigate a modified version of the PageRank algorithm which takes topic-level information into account.
+They use an scientific article data set where they manually label their data set with queries. 
+In this paper, we want to expand the framework which \cite{yang2009topic} have provided.
+But the data set which \cite{yang2009topic} uses is based on scientific articles, which includes citation to other similar articles.
+However, the citation scheme does not extent to other document types, such as news articles.
+
+In this paper, we want to investigate whether using a similarity measure based on the topics generated by a topic model can be used to PageRank.
+
+The combination of \gls{lda} and PageRank is interesting, since underlying document topics can yield more information than usually given by a single document.
+These topics can be seen as a set of documents sharing a specific topic. 
+
+When evaluating search engines, manual labeling of documents is usually used to measure the performance of models\cite{yang2009topic}\cite{Tang2008}.
+Since the topics, that is created by the \gls{lda}, is a distribution between of topics over documents, we can use these to evaluate a search algorithm's retrieval performance. 
```

Comment:
```suggestion
Since the topics, that are created by the \gls{lda}, are distributions of topics in documents, we can use these to evaluate a search algorithm's retrieval performance. 
```